handlers.go

```diff
@@ -2938,7 +2942,7 @@ func (s *Server) handleLoginUser(c echo.Context) error {
 		}
 	}
 
-	authToken, err := s.newAuthTokenForUser(&user, time.Now().Add(time.Hour*24*30), nil)
+	authToken, err := s.newAuthTokenForUser(&user, time.Now().Add(time.Hour*24*30), nil, "login")
```

Comment:
@alvin-reyes why do we create new API keys every time a user logs in?

Reply:
i believe this only happens if the user's cookie no longer has a token or has an expired token, so they get the login screen and have to login with their credentials. then upon successful login they get a new token and shouldn't hit the login endpoints anymore as long as their token is valid

Reply:
ok